### PR TITLE
Connect one tracker at a time text adjustment

### PR DESCRIPTION
**Pull request by ZRock35**
Fixes a small line of text to be more clear that one tracker should be connected at a time when setting up

### CODE REVIEW

**Review by Louka3000 — COMMENTED** [review]
Don't we already have a tip for turning the tracker on? Pointing this out as DIY tracker users may not want to turn them on...

**Comment by Louka3000** [comment]
slimevr_5GzgllYrXC

File: gui/public/i18n/en/translation.ftl [hunk 1]
```diff
@@ -602,8 +602,8 @@ onboarding-done-close = Close setup
 ## Tracker connection setup
 onboarding-connect_tracker-back = Go Back to Wi-Fi credentials
 onboarding-connect_tracker-title = Connect trackers
-onboarding-connect_tracker-description-p0 = Now onto the fun part, connecting all the trackers!
-onboarding-connect_tracker-description-p1 = Simply connect all that aren't connected yet, through a USB port.
+onboarding-connect_tracker-description-p0 = Now onto the fun part, connecting trackers!
+onboarding-connect_tracker-description-p1 = Connect one tracker at a time through a USB port and turn it on.
```
Comment (Louka3000) [inline gui/public/i18n/en/translation.ftl:606]:
```suggestion
onboarding-connect_tracker-description-p1 = Connect each tracker one at a time through a USB port.
```

**Review by Louka3000 — CHANGES_REQUESTED** [review]

File: gui/public/i18n/en/translation.ftl [hunk 2]
```diff
@@ -602,8 +602,8 @@ onboarding-done-close = Close setup
 ## Tracker connection setup
 onboarding-connect_tracker-back = Go Back to Wi-Fi credentials
 onboarding-connect_tracker-title = Connect trackers
-onboarding-connect_tracker-description-p0 = Now onto the fun part, connecting all the trackers!
-onboarding-connect_tracker-description-p1 = Simply connect all that aren't connected yet, through a USB port.
+onboarding-connect_tracker-description-p0 = Now onto the fun part, connecting trackers!
+onboarding-connect_tracker-description-p1 = Connect each tracker one at a time through a USB port.
```
Comment (ImUrX) [inline gui/public/i18n/en/translation.ftl:606]: Change the translation key so other languages are forced to retranslate it
```suggestion
onboarding-connect_tracker-description-p0-v1 = Now onto the fun part, connecting trackers!
onboarding-connect_tracker-description-p1-v1 = Connect each tracker one at a time through a USB port.
```

**Review by ImUrX — CHANGES_REQUESTED** [review]

**Review by ImUrX — APPROVED** [review]
lgtm

**Review by Louka3000 — APPROVED** [review]
Now learn Kotlin and typescript

**Pull request merged**